web/index.html

```diff
@@ -52,6 +52,14 @@ <h1><a href="https://zonebuilders.github.io/zonebuilder/articles/paper.html" tar
 				zoomOffset: -1
 			}).addTo(map);
 
+			// Add an optional raster layer for population
+			var worldpop = L.tileLayer('https://ogc.worldpop.org/geoserver/gwc/service/tms/1.0.0/wpGlobal%3Appp_2020@EPSG%3A900913@png/{z}/{x}/{-y}.png', {
+				tms: true,
+				opacity: 0.5,
+				maxNativeZoom: 11
```

Comment:
Nice, is it worth increasing that from 11 to a larger number, e.g. 17?

Reply:
No, the tiles are only available up to 11, based on manually testing and watching when the URL 404s. Setting it to 11 means when we zoom in further, we use the lower resolution tiles from zoom 11 and just stretch them. That's the best we can do without processing worldpop data into rasters ourself.

Reply:
:+1: